setuptools/tests/test_core_metadata.py

```diff
@@ -188,7 +188,7 @@ def test_read_metadata(name, attrs):
         ('requires', dist_class.get_requires),
```

Comment:
Line 313 still contains a reference to "ordered-set==3.1.1" (it's in an example, but may as well remove it)

Reply:
I'm +0.